src/SDKs/ContainerInstance/ContainerInstance.Tests/ContainerInstance.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>ContainerInstance.Tests</PackageId>
     <Description>ContainerInstance.Tests Class Library</Description>
     <AssemblyName>ContainerInstance.Tests</AssemblyName>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
```

Comment:
It is not necessary to modify the version number here. Simply set this to `1.0.0` since these won't be published